pyriksdagen/args.py

```diff
@@ -0,0 +1,163 @@
+"""
+Dry Argparse helper.
+
+"""
+from datetime import datetime
+from glob import glob
+from pyriksdagen.utils import (
+    get_data_location,
+    protocol_iterators,
+)
+import argparse
+import sys
+
+
+
+
+def record_parser(parser):
+    """
+    Take an argparse ArgumentParser object and populate standard arguments for working with riksdagen records;
+    args:
+        parser: parser
+    returns:
+        parser
+    """
+    parser.add_argument("-s", "--start",
+                        type=int,
+                        default=None,
+                        help="start year. If -s is set, -e must also be set. If -s and -e are explicitly set, all protocols in that range are processed. If -s and -e are unset and neither -p nor -l are set, the script will process all records from 1867 until the current year.")
+    parser.add_argument("-e", "--end",
+                        type=int,
+                        default=None,
+                        help="end year. If -e is set, -s must also be set. If -s and -e are explicitly set, all protocols in that range are processed. If -s and -e are unset and neither -p nor -l are set, the script will process all records from 1867 until the current year.")
+    parser.add_argument("-y", "--parliament-year",
+                        type=int,
+                        default=None,
+                        nargs='*',
+                        help="Parliament year, e.g. 1971 or 198384")
+    parser.add_argument("-R", "--records-folder",
+                        type=str,
+                        default=None,
+                        help="(optional) Path to records folder, defaults to environment var or `data/`")
+    parser.add_argument("-r", "--records",
+                        type=str,
+                        default=None,
```

Comment:
I usually use [] as the default argument for lists, would that work here? Or is None better?

Reply:
I don't know if it matters. It returns an empty list either way. I like None here because there's no confusion about the type what should be entered here... just a string or several strings